source/renderer/app/components/loading/syncing-connecting/SyncingConnectingStatus.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../themes/mixins/animations';
 
 .component {
-  bottom: 64px;
+  bottom: 65px;
```

Comment:
64px was actually correct. since row with icons is 24px, so 20 + 24 + 20 = 64
can we check that row with icons is 24px?

Reply:
✅

Reply:
you didn't push it))

Reply:
It's 25px. What needs to be changed?
![image](https://user-images.githubusercontent.com/1504716/144918174-433ee3aa-ef95-43a8-b2ae-26c4458f3c14.png)

Reply:
it should be 24px 
and now I see it's not vertically centered ((

Reply:
Updated to 24px